src/styles/typography/prose-scope.md.njk

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-prose-scope">
+# H1 heading
```

Comment:
I think you'll need to simulate markdown output by marking this up as HTML (without classes) for the scope to style. I think the markdown processor won't touch content inside HTML tags.

Reply:
Fixed